includes/constants/class-order-statuses.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Class Order_Statuses
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Constants;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+use MyCLabs\Enum\Enum;
+
+/**
+ * This class gives a list of all the possible order status constants.
+ *
+ * @psalm-immutable
+ */
+class Order_Statuses extends Enum {
```

Comment:
Also, to align with naming, please rename it to `Order_Status`

Reply:
I have updated it, and changed all respective references. Thanks!

802a8a4
67a1dc2
9a93972